### ISSUE TITLE
Make `forfeit` command to easily create forfeits for a bot that doesn't work

### ISSUE THREAD

**Issue by NicEastvillage** [RLBot/AutoLeaguePlay#20]

**Comment by tarehart** [comment]
Good idea! Another related issue is that it seems necessary at the moment to kill the ALP process to facilitate a FF. We should make sure that the solution doesn't require a hard kill, since that kills RLBot.exe, which tends to break the obs overlay, leave rendering junk on the screen, or crash rocket league.
